include/ebpf_nethooks.h

```diff
@@ -175,6 +175,26 @@ EBPF_HELPER(uint64_t, bpf_sock_addr_get_current_pid_tgid, (bpf_sock_addr_t * ctx
     ((bpf_sock_addr_get_current_pid_tgid_t)BPF_FUNC_sock_addr_get_current_pid_tgid)
 #endif
 
+#define SOCK_ADDR_GLOBAL_HELPER_FN_BASE 0xFFFF
+
+typedef enum
+{
+    BPF_FUNC_sock_addr_is_current_admin = SOCK_ADDR_GLOBAL_HELPER_FN_BASE + 1,
+} ebpf_sock_addr_global_helper_id_t;
+
+/**
+ * @brief Get current is_admin access (sock_addr global only).
+ *
+ * @param[in] ctx Pointer to bpf_sock_addr_t context.
+ *
+ * @returns a 32-bit integer containing the is_admin 0 or 1.
```

Comment:
Why is this different from how bpf_is_current_admin is documented?  It has:
```
 * @brief Get whether the current user is admin. In case of sock_addr attach types,
 * returns whether the user initiating the request is admin or not. In other
 * cases, returns whether the current thread user is admin or not.
 *
 * @param[in] ctx Context passed to the eBPF program.
 *
 * @retval 1 Is admin.
 * @retval 0 Is not admin.
 * @retval <0 An error occurred.

```

Reply:
You are right, I have removed the new code and tested with the contents from bpf_helper_defs.h